pkg/schema/tracker.go

```diff
@@ -54,9 +55,19 @@ var (
 
 // Tracker is used to track schema locally.
 type Tracker struct {
-	store kv.Storage
-	dom   *domain.Domain
-	se    session.Session
+	store     kv.Storage
+	dom       *domain.Domain
+	se        session.Session
+	toIndexes map[string]map[string]*ToIndexes
```

Comment:
I would like to put full downstream table schema info here so we can solve issues like #2108

Reply:
BTW, not sure if it's more convenient to flatten the two level map by changing from  `schema --> table --> tableInfo` to `unique_table_name --> tableInfo`

Reply:
OK, I will change downstream info to tableID -> table schema info and will enhance its expansibility.